.babelrc

```diff
@@ -2,5 +2,13 @@
   "presets": [
     "env",
     "react"
+  ],
+  "plugins": [
+    ["module-resolver", {
+      "alias": {
+        "colors": "./src/styleguide/colors.js",
+        "breakpoints": "./src/styleguide/breakpoints.js"
```

Comment:
what you think about create some groups like `styleguide/` and import something like colors using 
```js
import { blue } from 'styleguide/colors'
```
or
```js
import { mobileScreenSize } from 'styleguide/breakpoints'
```

Reply:
That was my first thought, and the first thing that I've made, but I preferred to take them as separated modules. But you prefer, I may change it.

Reply:
I prefer separated modules, its more clean and organized :smile:

Reply:
Okay